PhpSecInfo/Test/Application/php.php

```diff
@@ -31,12 +32,16 @@ function _retrieveCurrentValue()
     {
         $this->current_value = PHP_VERSION;
 
-        $url = 'https://php.net/releases/?serialize=1&version=7';
+        $url = "https://php.net/releases/?json=1&version=" . self::PHP_MAJOR_VERSION;
```

Comment:
Shouldn't we use php's `PHP_MAJOR_VERSION` instead? That would look for updates of PHP 8 if someone is already using it

Reply:
Didn't think of that possibility.